TypeScript compilation errors with v1.1.0

After updating to v1.1.0 of chartjs-plugin-zoom I can't build my typescript project because I get the following errors:

```
Error: node_modules/chartjs-plugin-zoom/types/index.d.ts:50:30 - error TS7006: Parameter 'chart' implicitly has an 'any' type.

50 export function getZoomLevel(chart): number;
                                ~~~~~


Error: node_modules/chartjs-plugin-zoom/types/options.d.ts:173:3 - error TS2411: Property 'x' of type 'ScaleLimits | undefined' is not assignable to string index type 'ScaleLimits'.

173   x?: ScaleLimits;
      ~


Error: node_modules/chartjs-plugin-zoom/types/options.d.ts:174:3 - error TS2411: Property 'y' of type 'ScaleLimits | undefined' is not assignable to string index type 'ScaleLimits'.

174   y?: ScaleLimits;
      ~



[ERROR] An error occurred while running subprocess ng.

        ng.cmd run app:build:production exited with exit code 1.
```

My version of Chart.js is v3.4.0

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -47,4 +47,4 @@ export function pan(chart: Chart, amount: PanAmount, scales?: Scale[], mode?: Up
 export function zoom(chart: Chart, amount: ZoomAmount, mode?: UpdateMode): void;
 export function zoomScale(chart: Chart, scaleId: string, range: ScaleRange, mode?: UpdateMode): void;
 export function resetZoom(chart: Chart, mode?: UpdateMode): void;
-export function getZoomLevel(chart): number;
+export function getZoomLevel(chart: Chart): number;

--- a/types/options.d.ts
+++ b/types/options.d.ts
@@ -169,10 +169,7 @@ export interface ScaleLimits {
 }
 
 export interface LimitOptions {
-  // Default horizontal and vertical scale limits
-  // x?: ScaleLimits;
-  // y?: ScaleLimits;
-  // Optional additional scale limits, indexed by the scale's ID (key)
+  // Scale limits, indexed by the scale's ID (key) or by axis (x/y)
   [axisId: string]: ScaleLimits;
 }
 

--- a/types/options.d.ts
+++ b/types/options.d.ts
@@ -170,8 +170,8 @@ export interface ScaleLimits {
 
 export interface LimitOptions {
   // Default horizontal and vertical scale limits
-  x?: ScaleLimits;
-  y?: ScaleLimits;
+  // x?: ScaleLimits;
+  // y?: ScaleLimits;
   // Optional additional scale limits, indexed by the scale's ID (key)
   [axisId: string]: ScaleLimits;
 }

--- a/types/tests/exports.ts
+++ b/types/tests/exports.ts
@@ -25,6 +25,11 @@ const chart = new Chart('id', {
             min: 1,
             max: 2,
             minRange: 1
+          },
+          y2: {
+            min: 10,
+            max: 20,
+            minRange: 5
           }
         },
         pan: {

--- a/types/tests/tsconfig.json
+++ b/types/tests/tsconfig.json
@@ -3,6 +3,7 @@
     "target": "ES6",
     "moduleResolution": "Node",
     "alwaysStrict": true,
+    "strict": true,
     "noEmit": true
   },
   "include": [

